### ISSUE TITLE
How to Generate an SBOM with Syft tutorial

### ISSUE THREAD

**Issue by sheesh** [chainguard-dev/edu#460]

**Comment by viveksahu26** [comment]
Hey @sheesh , can I take this.

**Comment by smythp** [comment]
@viveksahu26 Still interested in this? I'll review the PR if you want to take a stab at it. No worries if your life has moved on :)

**Comment by viveksahu26** [comment]
@smythp  Certainly, life has moved on to S3C fields. Assign it to me, and I'll try to work on it over the weekends.

**Comment by smythp** [comment]
Congrats on the role. Here's a link to a recent intro tutorial on Grype on Academy that might provide some inspiration:

https://edu.chainguard.dev/chainguard/chainguard-images/working-with-images/scanners/grype-tutorial/

No worries if it turns out you can't do it, we'll just put this issue back in the icebox 🧊

**Comment by SharpRake** [comment]
Closing, as we have this documented here: https://edu.chainguard.dev/open-source/sigstore/cosign/how-to-sign-an-sbom-with-cosign/#generating-an-sbom-with-syft
